Product_Landing_Page/css/style.css

```diff
@@ -0,0 +1,327 @@
+}*/
+
+header{
+  display: flex;
+  min-height: 75px;
+  padding: 0px 20px;
+  position: fixed;
+  top: 0;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #eee;
+}
+
+@media (max-width:600px) {
+
+  header{
+    flex-wrap: wrap;
+  }
+
+}
+
+#page-logo{
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  text-align: center;
+  width:60vw
+
+}
+
+
+#page-logo > img {
+  width: 100%
+  height: 100%;
+  max-width: 300px;
+  padding-left: 15px
+}
+
+
+@media (max-width:600px){
+  #page-logo{
+    margin-top: 10px;
+    width: 100%;
+    position: relative;
+    align-content: center;
+  }
+  #page-logo > img{
+    margin-left: auto;
+    margin-right: auto;
+  }
+}
+/*@media (max-width: 600px) {
+  #page-logo {
+    justify-content: center;
+    align-items: center;
+  }
+}*/
+
+
+
+#navigational-bar{
+  width: 35vw;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+
+}
+
+@media (max-width:800px){
+  #navigational-bar{
+    margin-top: 10px;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    padding: 0 50px;
+
+  }
+#navigational-bar li {
+    padding-bottom: 5px;
+  }
+}
+/*
+@media (max-width: 600px) {
+  header {
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+  }
+}*/
+
+/*@media (max-width: 600px) {
+    #page-logo > img {
+      align-items: center;
+      justify-content: center;
+
+    }
+}*/
+
+#emailform {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  height: 200px;
+  margin-top: 50px;
+}
+
+#emailform,h2 {
+  margin-bottom: 20px;
+
+}
+
+#emailform input[type="email"] {
+  max-width: 275px;
+  width: 100%;
+  padding: 5px;
+}
+
+#emailform input[type="submit"] , .btn{
+  max-width: 150px;
+  width: 100%;
+  height: 30px;
+  margin: 15px 0;
+  border: 0;
+  background-color: #f1c40f;
+}
+
+#emailform input[type="submit"]:hover , .btn:hover{
+  background-color: orange;
+  transition: background-color 1s;
+}
+
+@media (max-width: 650px) {
+  #emailform {
+    margin-top: 120px;
+  }
+}
+
+.features-section
+{
+  max-width: 1000px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.feature{
+  display: flex;
+  flex-direction: row;
+}
+
+.icons{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 125px;
+  width: 20vw;
+
+}
+
+.description{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 125px;
+  width: 80vw;
+  padding: 5px;
+}
+
+@media (max-width:600px) {
+  .icons{
+    display: none;
+  }
+  .description{
+    width: 100%;
+    text-align: center;
+    padding: 0;
+    height: 150px;
+  }
+}
+
+#video1{
+  display: none;
+}
+
+.video-section{
+  max-width: 1000px;
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+}
+
+.video-section > iframe{
+  max-width: 600px;
```

Comment:
Please add class to iframe. And use that to change styling.

Reply:
done